ddtrace/debugging/_debugger.py

```diff
@@ -246,6 +248,12 @@ def __init__(self, tracer=None):
         )
 
         # Register the debugger with the RCM client.
+        if asbool(os.environ.get("DD_REMOTE_CONFIGURATION_ENABLED", True)) is False:
+            os.environ["DD_REMOTE_CONFIGURATION_ENABLED"] = "true"
+            log.info(
+                "Dynamic Instrumentation feature is enabled but Remote Config is explicit disabled."
```

Comment:
We don't expect customers to be bothered by this internal detail so we can perhaps keep the message short, e.g.
```suggestion
                "Disabled Remote Configuration enabled by Dynamic Instrumentation."
```